### PR TITLE
Feature: QR Code of the URL added

### LINKED ISSUE #22: Providing QR codes
Provide the QR code for the generated short-URLs so that the user can have the QR code image. QR codes promote **interaction** and **engagement** through the mobile phone. It can be used by businesses to redirect users to their products. This is useful as short-URLs are easy and faster to scan in comparison to long URLs.
**Refer**: API for generating [QR codes](http://goqr.me/api/)

### PR DESCRIPTION
**Pull request by arnabsen1729**
Fixed #22 

## Before
image

## After 
image

### CODE REVIEW

File: public/js/index.js [hunk 1]
```diff
@@ -66,3 +67,9 @@ function ClipBoard(result) {
 	navigator.clipboard.writeText(result);
 	document.getElementById("copy").innerHTML='Copied'
 }
+
+function updateQRImage(shortenedUrl){
+	let QRImg = document.getElementById('QRImage');
```
Comment (arjungarg07) [inline public/js/index.js]: It's a good practice to use const. So replace let with const

**Review by arjungarg07 — CHANGES_REQUESTED** [review]

File: public/js/index.js (hunk 1 above)
Comment (arnabsen1729) [reply public/js/index.js]: Yeah, you are right!

Comment (arnabsen1729) [reply public/js/index.js]: Fixed in https://github.com/arjungarg07/Brief-Me/pull/25/commits/96f2efcd57ac7bbde8711562f84f32763563df19

**Review by arjungarg07 — APPROVED** [review]

**Comment by arjungarg07** [comment]
Thanks!

**Pull request merged**